english/marshmallow-attack/marsh-attack.md

```diff
@@ -6,7 +6,9 @@ reaches for the lighter you quickly jab the fork into the
 
 white fluffy hand. The marshmallow pulls back its hand and
 
-a giant gaping mouth appears and starts to scream.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.